CHANGELOG.md

```diff
@@ -213,7 +213,7 @@ Other minor additions
   ```agda
   *-pres-∣ : o ∣ m → p ∣ n → o * p ∣ m * n
   ```
-  
+
```

Comment:
Why is your `fix-whitespace` picking up whitespace in the CHANGELOG when it's supposed to be ignored?

https://github.com/agda/agda-stdlib/blob/911b52d54d39939a78ba63a387e30cc90ad64163/fix-whitespace.yaml#L12

Reply:
My emacs is configured to automatically fix whitespace issues. :x